backup-loop.sh

```diff
@@ -300,9 +301,9 @@ restic() {
 
 rclone() {
   _find_old_backups() {
-    command rclone lsf --format "tp" "${RCLONE_REMOTE}:${DEST_DIR}" | grep ${BACKUP_NAME} | awk \
+    command rclone lsf --format "tp" "${RCLONE_REMOTE}:${RCLONE_DEST_DIR}" | grep ${BACKUP_NAME} | awk \
```

Comment:
Should it perhaps assert `RCLONE_DEST_DIR` is non-empty value?

Reply:
Nah, some types of remotes are writable at the root; an empty `RCLONE_DEST_DIR` is valid in that case.